src/orquestra/sdk/_base/_driver/_client.py

```diff
@@ -882,11 +882,11 @@ def get_workflow_project(self, wf_run_id: _models.WorkflowRunID) -> ProjectRef:
         Gets the status of a workflow run from the workflow driver
 
         Raises:
-            InvalidWorkflowRunID: see the exception's docstring
-            WorkflowRunNotFound: see the exception's docstring
-            InvalidTokenError: see the exception's docstring
-            ForbiddenError: see the exception's docstring
-            UnknownHTTPError: see the exception's docstring
+            orquestra.sdk._base._driver._exceptions.InvalidWorkflowRunID:
+            orquestra.sdk._base._driver._exceptions.WorkflowRunNotFound:
+            orquestra.sdk._base._driver._exceptions.InvalidTokenError:
+            orquestra.sdk._base._driver._exceptions.ForbiddenError:
+            orquestra.sdk._base._driver._exceptions.UnknownHTTPError:
```

Comment:
Nitpick: Is the `:` needed in those lists? This applies to all the lies changed in docstring under raises:

Reply:
Just checked and you're right! Removing the trailing `:` doesn't produce warnings. I'll update the PR.